src/Entity/Lobby.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Entity;
+
+use ApiPlatform\Core\Annotation\ApiResource;
+use App\Repository\LobbyRepository;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ApiResource(security="is_granted('ROLE_USER','ROLE_ADMIN')")
```

Comment:
il faudrait plutôt un truc dans ce goût là.
```suggestion
 * @ApiResource(security="is_granted('ROLE_USER') or is_granted('ROLE_ADMIN')")
```

Bonus, tu peux même définir des hiérarchies de role
https://symfony.com/doc/current/security.html#hierarchical-roles

Reply:
Ok je ne connaissais pas, intéressant ! Je suis une jeune pousse sur symfony et en PHP en général, j'étais un peu perdu ^^